Client/Frontend/Settings/BraveShieldsAndPrivacySettingsController.swift

```diff
@@ -45,6 +55,32 @@ class BraveShieldsAndPrivacySettingsController: TableViewController {
       manageWebsiteDataSection,
       otherSettingsSection,
     ]
+    
+    // Listen to changes on filter lists so we know we need to reload the section
+    filterListsSubscription = FilterListResourceDownloader.shared.$filterLists
+      .receive(on: DispatchQueue.main)
+      .sink { filterLists in
+        let filterList = filterLists.first(where: { $0.uuid == FilterListResourceDownloader.cookieConsentNoticesUUID })
+        let isEnabled = filterList?.isEnabled ?? false
```

Comment:
`let isEnabled = filterList?.isEnabled == true` should be easier to read

Reply:
rabbit